.github/workflows/go-update.yml

```diff
@@ -44,13 +44,13 @@ jobs:
       - name: Setup Python and pip
         uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
         with:
-          python-version: 3.x
+          python-version: "3.12"
```

Comment:
Required because the latest major version of Python was bumped to 3.13 and the developer tooling, currently, requires 3.12.

Reply:
Thanks !